src/Console/ModelsCommand.php

```diff
@@ -439,10 +439,17 @@ protected function getPropertiesFromTable($model)
      */
     protected function getPropertiesFromMethods($model)
     {
-        $methods = get_class_methods($model);
+        $reflection = new ReflectionClass($model);
+        $methods = $reflection->getMethods();
         if ($methods) {
-            sort($methods);
-            foreach ($methods as $method) {
+            usort($methods, function ($a, $b) {
+                if ($a->name == $b->name) {
+                    return 0;
+                }
+                return ($a->name < $b->name) ? -1 : 1;
```

Comment:
I belief this can be simplified withL
```suggestion
                return $a->name <=> $b->name;
```